### PR TITLE
[new release] owl-zoo, owl-top, owl-base, owl and owl-plplot (1.0.1)

### PR DESCRIPTION
**Pull request by jzstark**
OCaml Scientific and Engineering Computing - Zoo

- Project page: <a href="https://github.com/owlbarn/owl">https://github.com/owlbarn/owl</a>
- Documentation: <a href="https://owlbarn.github.io/">https://owlbarn.github.io/</a>

##### CHANGES:

* Add eighth-order finite difference approximation
* Fix bug in Jacobian with different input/output dimensions (owlbarn/owl#557)
* Fix bug in nested grads (owlbarn/owl#558)
* Update to ocamlformat.0.16.0 (thanks @gpetiot owlbarn/owl#556)
* Add get_fancy to AD
* Check input and output type of `diff` operation
* Fix bug in `build_info` of aiso pattern in AD
* Implement split forward mode and check max tag of `build_info` 
* Add initial implementation of fft2 and ifft2
* Add nonsymmetric qs suppport for continuous and discrete time lyapunov gradients
* Improve `care` and `dare` operations in AD
* Improve forward mode efficiency for sylv, clyap and dlyap operations in AD

### CODE REVIEW

**Comment by mseri** [comment]
- The linking errors on ubuntu-18 are a known issue, there one needs to rebuild a custom openblas. Was fixed in ubuntu 20. 
- I am not sure 32 bits systems are supported, @jzstark do you know? If so, the error on i386 needs to be looked at
- PPC is not supported AFAIK.
- ARM support is on its way, but not there yet.
- All other errors are for missing external libraries, so can be ignored

File: packages/conf-openblas/conf-openblas.0.2.1/opam [hunk 1]
```diff
@@ -31,6 +31,10 @@ depexts: [
   ["openblas"] {os = "macos" & os-distribution = "homebrew"}
   ["openblas"] {os = "freebsd"}
 ]
+x-ci-accept-failures: [
```
Comment (mseri) [inline packages/conf-openblas/conf-openblas.0.2.1/opam:34]: Neat!

File: packages/conf-plplot/conf-plplot.1/opam [hunk 2]
```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
```
Comment (mseri) [inline packages/conf-plplot/conf-plplot.1/opam:1]: Thanks. I had meant to do this for a while and always procrastinated

File: packages/conf-openblas/conf-openblas.0.2.1/opam (hunk 1 above)
Comment (kit-ty-kate) [reply packages/conf-openblas/conf-openblas.0.2.1/opam:34]: Yes, I just merged this feature yesterday (https://github.com/ocurrent/opam-repo-ci/pull/62), sorry I forgot to tell you about it

Comment (mseri) [reply packages/conf-openblas/conf-openblas.0.2.1/opam:34]: Nothing to be sorry about, don't worry. This is a fantastic news, I am glad we have it finally 🎉

**Comment by kit-ty-kate** [comment]
Ok let's stop there. At least all the new packages and their revdeps pass now \o/

cc @mkur @hcarty for the new `conf-plplot` and `conf-nlopt` packages.

Thanks!

**Pull request merged**